tracer/test/test-applications/regression/AutomapperTest/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM mcr.microsoft.com/dotnet/core/runtime:2.1-stretch-slim AS base
-ARG TRACER_VERSION=2.9.0
+ARG TRACER_VERSION=2.8.0
```

Comment:
I see that this change is needed, as it using the release version, but it kinda looks like it should be the version we just build and run using the standard mechanisms.

(Although, to be setup like this, it's obviously a special case, and I don't know the history, so probably not worth changing).

Reply:
Yep, I agree, needs looking into (I'll take a look in a separate PR)

Reply:
Thanks for flagging - I've looked into it and the Dockerfile doesn't seem necessary and is completely broken anyway AFAICT https://github.com/DataDog/dd-trace-dotnet/pull/2771